cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt

```diff
@@ -435,14 +435,19 @@ private class CampaignManagerImpl(
                 ?.let { true }
                 ?: false
 
-            val res = (isNewUser || ccpaToBeCompleted || gdprToBeCompleted)
+            val usNatToBeCompleted: Boolean = spConfig.campaigns.find { it.campaignType == CampaignType.USNAT }
+                ?.let { true }
+                ?: false
+
+            val res = (isNewUser || ccpaToBeCompleted || gdprToBeCompleted || usNatToBeCompleted)
```

Comment:
we don't need parentheses here

Reply:
I don’t feel comfortable removing parentheses, it is safer to keep them